src/controllers/notificationController.js

```diff
@@ -34,3 +34,19 @@ const { Notification } = Model;
         return respondWithWarning(res, statusCode.internalServerError, 'Server Error');
     });
 }
+
+export const getAllUserNotification = async (req, res) => {
+  // de-structure id out of req.auth, then rename id to userId
+  const { id: userId } = req.auth;
+
+  try {
+    const notifications = await getAllNotifications(userId);
+    if (notifications.length < 1) {
```

Comment:
`if (notifications.length < 1)` can be refactored to something cleaner `if (!notifications.length)`